proposals/dataoperation/v1.0_dataflow_affinity/v1.0_dataflow_affinity_en.md

```diff
@@ -0,0 +1,227 @@
+# The affinity between Data Operation Pods in DataFlow
+
+## Motivation
+Currently, Fluid supports DataFlow, which allows multiple Data Operations to be executed sequentially. However, different data operation pods may run on different nodes, resulting in overall low performance:
+
+- If the subsequent Data Operations can run on the same ***node/zone/region*** as the previous Data Operations, it may improve performance.
```

Comment:
it may improve performance -》 it may improve data consuming performance

Reply:
ok, fixed.